src/bundle/Resources/views/fieldtypes/edit/ezdate.html.twig

```diff
@@ -7,7 +7,10 @@
             </svg>
         </button>
     </div>
-    {% set action_type = app.request.attributes.get('_route') is same as('ez_content_draft_edit') ? 'edit' : 'create' %}
+    {% set action_type =
+        app.request.attributes.get('_route') is same as('ez_content_draft_edit') or
+        app.request.attributes.get('_route') is same as('ezplatform.content.translate') ? 'edit' : 'create'
+    %}
```

Comment:
You can use `in` operator here:

```
app.request.attributes.get('_route') in ['ez_content_draft_edit', 'ezplatform.content.translate']
```

but TBH this logic should be placed in form value mapper or / Form type `buildView` method rather then in Twig template